tests_integration_with_local_chain/markets/omen/test_omen.py

```
@@ -405,8 +459,8 @@
         outcome=True,
         amount=bet_amount,
         omen_auto_deposit=True,
+        api_keys=keys,
```

Ruff

Undefined name `keys`

(F821)